cockatrice/src/decklistmodel.cpp

```diff
@@ -311,9 +311,9 @@ void DeckListModel::sortHelper(InnerDecklistNode *node, Qt::SortOrder order)
         const int fromRow = sortResult[i].first;
         const int toRow = sortResult[i].second;
         AbstractDecklistNode *temp = node->at(toRow);
-        for (int j = columnCount(); j; --j) {
-            from << createIndex(fromRow, 0, temp);
-            to << createIndex(toRow, 0, temp);
+        for (int j = 0; j < columnCount(); ++j) {
```

Comment:
While this won't have much effect in this instance, I think calling `columnCount()` every loop is considered bad practice.